ngraph/core/include/openvino/core/variant.hpp

```diff
@@ -95,7 +96,5 @@ inline std::shared_ptr<Variant> make_variant(const wchar_t (&s)[N]) {
     return std::dynamic_pointer_cast<VariantImpl<std::wstring>>(std::make_shared<VariantWrapper<std::wstring>>(s));
 }
 #endif
-
-using RTMap = std::map<std::string, std::shared_ptr<Variant>>;
 using VariantVector = std::vector<std::shared_ptr<Variant>>;
```

Comment:
@GlebKazantaev @ilyachur should we also replace `VariantVector` usage with `std::vector<ov::Any>`?
It's used in frontends and variant is not appropriate place there

Reply:
will be done in other PR